src/libcollections/vec.rs

```diff
@@ -989,6 +989,46 @@ impl<T> Vec<T> {
             result
         }
     }
+
+    /// Splits the collection into two at the given index.
+    ///
+    /// Returns a newly allocated `Self`. `self` contains elements `[0..at]`,
+    /// and the returned `Self` contains elements `[at ..]`.
```

Comment:
I'm a bit nervosd about this since this is accurate *Rust* notation, but confusing as *math* notation.

*probably* fine as is, though.